_maps/shuttles/emergency_saltmine.dmm

```diff
@@ -0,0 +1,944 @@
+"bs" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/reinforced,
+/obj/item/reagent_containers/glass/beaker/waterbottle/large,
+/obj/structure/window/plasma/reinforced{
+	dir = 1
+	},
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bt" = (
+/obj/effect/decal/cleanable/salt,
+/obj/item/grown/bananapeel,
+/obj/structure/window/plasma/reinforced{
+	dir = 1
+	},
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bu" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/reinforced,
+/obj/item/clothing/head/collectable/wizard,
+/obj/structure/window/plasma/reinforced{
+	dir = 1
+	},
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bv" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/glass,
+/obj/item/clothing/head/kitty/genuine,
+/obj/structure/window/plasma/reinforced{
+	dir = 1
+	},
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bw" = (
+/obj/effect/decal/cleanable/salt,
+/obj/item/grown/bananapeel,
+/obj/structure/window/plasma/reinforced{
+	dir = 1
+	},
+/obj/item/device/assembly/flash,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bx" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/reinforced,
+/obj/structure/window/plasma/reinforced{
+	dir = 1
+	},
+/obj/item/toy/plush/lizardplushie,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"by" = (
+/obj/effect/decal/cleanable/salt,
+/turf/closed/wall/mineral/titanium,
+/area/shuttle/escape)
+"bz" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/reinforced,
+/obj/item/clothing/suit/wizrobe,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bA" = (
+/obj/structure/table/glass,
+/obj/effect/decal/cleanable/salt,
+/obj/item/clothing/head/rabbitears,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bB" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/reinforced,
+/obj/item/stack/sheet/animalhide/lizard,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bC" = (
+/obj/structure/table/reinforced,
+/obj/effect/decal/cleanable/salt,
+/obj/item/bikehorn,
+/obj/item/organ/eyes/moth,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bD" = (
+/obj/effect/decal/cleanable/salt,
+/turf/open/floor/fakepit,
+/area/shuttle/escape)
+"bE" = (
+/obj/structure/table/glass,
+/obj/effect/decal/cleanable/salt,
+/obj/item/storage/firstaid/fire,
+/turf/open/floor/fakepit,
+/area/shuttle/escape)
+"bF" = (
+/obj/machinery/light,
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/reinforced,
+/obj/item/gun/magic/wand/resurrection/inert,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bG" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/glass,
+/obj/item/clothing/head/rabbitears,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bH" = (
+/obj/effect/decal/cleanable/salt,
+/obj/structure/table/reinforced,
+/obj/item/reagent_containers/food/drinks/bottle/lizardwine,
+/turf/open/floor/fakespace,
+/area/shuttle/escape)
+"bI" = (
+/obj/effect/decal/cleanable/salt,
+/turf/open/chasm,
```

Comment:
THIS ONE WILL B A FAN FAVORITE LOOOOOOL

(you shouldn't place the salt and the real chasm on the same tile the salt will always fall into the chasm so it'll be like it didn't exist)

Reply:
Are you sure about that

Reply:
no

Reply:
unless you're talking about this being a fan favorite, which in that case yes